pkg/virt-handler/vm_test.go

```diff
@@ -794,139 +794,130 @@ var _ = Describe("VirtualMachineInstance", func() {
 			))
 		})
 
-		It("should add access credential synced condition when credentials report success", func() {
-			vmi := api2.NewMinimalVMI("testvmi")
-			vmi.UID = vmiTestUUID
-			vmi.ObjectMeta.ResourceVersion = "1"
-			vmi.Status.Phase = v1.Running
-			vmi = addActivePods(vmi, podTestUUID, host)
+		Context("access credential synced condition", func() {
+			var (
+				vmi    *v1.VirtualMachineInstance
+				domain *api.Domain
+			)
 
-			domain := api.NewMinimalDomainWithUUID("testvmi", vmiTestUUID)
-			domain.Status.Status = api.Running
-			domain.Spec.Metadata.KubeVirt.AccessCredential = &api.AccessCredentialMetadata{
-				Succeeded: true,
-				Message:   "",
+			BeforeEach(func() {
+				vmi = libvmi.New(
+					libvmi.WithNamespace(k8sv1.NamespaceDefault),
+					libvmi.WithResourceMemory("8192Ki"),
```

Comment:
That has to be the tiniest VM I've ever seen. :) Do we even need to set this at all?

Reply:
True, it was not needed.